app/addons/documents/index-results/index-results.components.react.jsx

```diff
@@ -28,10 +36,118 @@ function (app, FauxtonAPI, React) {
     }
   });
 
+  var ResultsScreen = React.createClass({
+
+    onDoubleClick: function (id, doc) {
+      FauxtonAPI.navigate(doc.url);
+    },
+
+    getUrlFragment: function (url) {
+      if (this.props.hasReduce) {
+        return null;
+      }
+
+
+      return (
+        <a href={url}>
+          <i className="fonticon-pencil"></i>
+        </a>);
+    },
+
+    getDocumentList: function () {
+      return _.map(this.props.results, function (doc) {
+        return (
+         <Components.Document
+           key={doc.id}
+           doc={doc}
+           onDoubleClick={this.onDoubleClick}
+           keylabel={doc.keylabel}
+           docContent={doc.content}
+           checked={this.props.isSelected(doc.id)}
+           docChecked={this.props.docChecked}
+           docIdentifier={doc.id} >
+           {this.getUrlFragment('#' + doc.url)}
+         </Components.Document>
+       );
+     }, this);
+    },
+
+    render: function () {
+      var classNames = 'view';
+
+      if (this.props.isDeleteable) {
+        classNames += ' show-select';
+      }
+
+      return (
+      <div className={classNames}>
+        <div id="doc-list">
+          <ReactCSSTransitionGroup transitionName={'slow-fade'}>
```

Comment:
can we make that a very fast fade? maybe 500ms

it irritates a bit when i want to go two pages forward